### ISSUE TITLE
Question on OpenMM Capability

### ISSUE THREAD

**Issue by LobsterMan123** [openmm/openmm#4619]
Would someone please tell me if OpenMM can do something called Gaussian accelerated molecular dynamics?  If yes, is this particularly difficult to set up/do?  I am totally new to OpenMM so I'd like to get an idea of the learning curve in order to use the software.

If OpenMM does not do Gaussian accelerated molecular dynamics, can anyone recommend a package that can do it?  Thanks.

LM

**Comment by peastman** [comment]
Yes, there's an implementation of it at https://github.com/MiaoLab20/GaMD-OpenMM.

**Comment by LobsterMan123** [comment]
Thank you very much.  Lastly, do I obtain files such as topology and parameter files from the MacKerell website (https://mackerell.umaryland.edu/charmm_ff.shtml)?

As a previous NAMD user, this was where I would obtain those files for NAMD, but I wasn't sure if that also applied to OpenMM.  Thanks again.

LM

**Comment by peastman** [comment]
OpenMM can run simulations starting from a variety of file types.  It can read CHARMM files, if that's what you want, but also Amber or Gromacs files, and it can also parameterize systems just starting from a PDB file.  See [the user guide](http://docs.openmm.org/latest/userguide/application/02_running_sims.html) for a description of how to run simulations.
